### ISSUE TITLE
Attempting small datasets

### ISSUE THREAD

**Issue by cnh-boom** [fistyee/MixPro#4]
I am trying to run this code using CIFAR-10, how should I operate it

**Comment by fistyee** [comment]
If you are training based on ViT, due to the default 224x224 image size of config, when using CIFAR-10 data, you only need to modify the dataloader to first resize the CIFAR-10 image to 224x224 , and use the augmentation operations (randaug, cutout...) in imagenet's dataloader, and after that, use the default pipeline for training. If you need other image sizes, you can adjust patch_size and mask_size in config, patch_size needs to be divisible by image size, and mask_size is a multiple of patch_size. Note that training ViT on such small data may not converge if you don't use pre-training and in small size. If you are training based on ResNet, simply copy MaskMix (int utlis.py) into your project, adjust the mask_size, others operation similar to mixup.

**Comment by cnh-boom** [comment]
Thank you very much for your reply. I have another question now. After reading the paper "MIXPRO: DATA AUGMENTATION With MASKMIX AND PROGRESSIVE ATTENTION LABELING FOR VISIONTRANSFORMER", I don't know how the mixing samples label y is used for data augmentation.

**Comment by fistyee** [comment]
Simply put, you can get the mixing sample label y from the (mask area ratio of a) or (mask area ratio of b) for the mixed images a and b. Also, you can use an attention-based progressive mixing label, according to the code in the main. py.
